security-center/snippets/system-test/v2/bigQueryExport.test.js

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {SecurityCenterClient} = require('@google-cloud/security-center').v2;
+const {assert} = require('chai');
+const {execSync} = require('child_process');
+const exec = cmd => execSync(cmd, {encoding: 'utf8'});
+const {describe, it, before} = require('mocha');
+const {BigQuery} = require('@google-cloud/bigquery');
+
+const organizationId = process.env.GCLOUD_ORGANIZATION;
+const projectId = process.env.GOOGLE_SAMPLES_PROJECT;
+const location = 'global';
+const bigquery = new BigQuery();
+
+async function cleanupDatasets() {
+  const [datasets] = await bigquery.getDatasets();
+  for (const dataset of datasets) {
+    if (dataset.id.startsWith('my_new_dataset')) {
+      console.log(`Deleting dataset: ${dataset.id}`);
+      await bigquery.dataset(dataset.id).delete({force: true});
+    }
+  }
+}
+
+async function cleanupBigQueryExports(client) {
+  const [exports] = await client.listBigQueryExports({
+    parent: client.organizationLocationPath(organizationId, location),
+  });
+  for (const exportData of exports) {
+    console.log(`Deleting BigQuery export: ${exportData.name}`);
+    await client.deleteBigQueryExport({name: exportData.name});
+  }
+}
+
+let dataset;
+
+async function createDataset() {
+  const datasetId = 'my_new_dataset';
```

Comment:
issue: Sample testing covers a large number of samples, and the resources often need to be recognizable by name for what they cover. 

suggestion: Prefix test resource as `securitycenter_` instead of `my_new_`, especially given this is a test.

Reply:
Agreed. Addressed.